xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/service/OkCoinFuturesAccountService.java

```diff
@@ -40,4 +44,8 @@ public String requestDepositAddress(Currency currency, String... args) throws IO
     throw new NotAvailableFromExchangeException();
   }
 
```

Comment:
I suggest adding a `TradeHistoryParams createFundingHistoryParams();` method that creates a default implementation, analogous to  `createTradeHistoryParams()` and `createOpenOrdersParams()` in `interface TradeService`. Only this way can users use the generic API without exchange-specific code (which is the purpose of the generic API).

Default implementations in exchanges with no support for this may return null.